PovioKit/Classes/Utilities/Throttler/Throttler.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  Throttler.swift
+//  PovioKit
+//
+//  Created by DK on 15/05/2018.
+//  Copyright © 2018 Povio Labs. All rights reserved.
+//
+
+import Foundation
+
+public class Throttler {
+	private let queue: DispatchQueue
+	private var job: DispatchWorkItem?
+	public var delay: TimeInterval
+	
+	public init(queue: DispatchQueue = .main, delay: TimeInterval = 0.5) {
```

Comment:
Can we use `DispatchTimeInterval` instead of `TimeInterval`? That way we can precisely define the time unit whereas now we don't know that value represents.